src/Compilers/CSharp/Portable/Operations/CSharpOperationNodes.cs

```diff
@@ -817,18 +822,14 @@ internal sealed class CSharpLazyInvocationOperation : LazyInvocationOperation
 
         protected override IOperation CreateInstance()
         {
-            BoundExpression receiver;
-            switch (_invocableExpression)
```

Comment:
>switch (_invocableExpression) [](start = 12, length = 29)

Please revert refactoring, simply add new case that you now want to handle. It should be obvious to a reviewer that other cases are not touched and the best way to achieve this is to not touch them.